include/baresip.h

```diff
@@ -52,7 +52,9 @@ struct vidsz;
 enum answermode {
 	ANSWERMODE_MANUAL = 0,
 	ANSWERMODE_EARLY,
-	ANSWERMODE_AUTO
+	ANSWERMODE_AUTO,
+	ANSWERMODE_EARLY_VIDEO,
+	ANSWERMODE_EARLY_AUDIO,
```

Comment:
can you switch the order of "video" and "audio" ?

just like the SDP the audio always comes first :)